modules/liveIntentIdSystem.js

```diff
@@ -171,6 +171,14 @@ export const liveIntentIdSubmodule = {
         result.uid2 = { 'id': value.uid2 }
       }
 
+      if (value.bidswitch) {
+        result.bidswitch = { 'id': value.bidswitch }
+      }
+
+      if (value.medianet) {
+        result.medianet = { 'id': value.medianet }
+      }
+
```

Comment:
For the reviewer: is there a better approach to expose multiple ids via a single user id module where each of the ids has its own config to create an eid out of it?